Index: llembeddedbrowserwindow.cpp
===================================================================
--- llembeddedbrowserwindow.cpp	(revision 26)
+++ llembeddedbrowserwindow.cpp	(working copy)
@@ -111,7 +111,7 @@
 	mStatusText( "" ),
 	mClickHref( "" ),
 	mClickTarget( "" ),
-	mNoFollowScheme( "secondlife://" ),
+	mNoFollowScheme( "meta7://" ),
 	mBkgRed( 0xff ),
 	mBkgGreen( 0xff ),
 	mBkgBlue( 0xff ),
Index: tests/ubrowser/ubrowser.cpp
===================================================================
--- tests/ubrowser/ubrowser.cpp	(revision 26)
+++ tests/ubrowser/ubrowser.cpp	(working copy)
@@ -115,7 +115,7 @@
 	gInstance = this;
 
 	// build list of home urls - first used for most things, rest used on a cube
-	mHomeUrl[ 0 ] = "http://secondlife.com";
+	mHomeUrl[ 0 ] = "http://meta7.com";
 	mHomeUrl[ 1 ] = "http://google.com";
 	mHomeUrl[ 2 ] = "http://mozilla.org";
 	mHomeUrl[ 3 ] = "http://yahoo.com";
@@ -124,8 +124,7 @@
 
 	// list of bookmarks that demonstrate cool stuff!
 	mBookmarks.push_back( std::pair< std::string, std::string >( "Select a bookmark", "" ) );
-	mBookmarks.push_back( std::pair< std::string, std::string >( "Second Life Home Page", "http://www.secondlife.com" ) );
-	mBookmarks.push_back( std::pair< std::string, std::string >( "Linden Lab Home Page", "http://www.lindenlab.com" ) );
+	mBookmarks.push_back( std::pair< std::string, std::string >( "meta7 Home Page", "http://www.meta7.com/" ) );
 	mBookmarks.push_back( std::pair< std::string, std::string >( "Boing Boing", "http://www.boingboing.net" ) );
 	mBookmarks.push_back( std::pair< std::string, std::string >( "Canvas Paint (DHTML version of MS Paint)", "http://www.canvaspaint.org" ) );
 	mBookmarks.push_back( std::pair< std::string, std::string >( "Canvascape", "http://www.abrahamjoffe.com.au/ben/canvascape/" ) );
